.gitignore

```diff
@@ -1,5 +1,6 @@
 *.gem
 *.rbc
+*.lock*
```

Comment:
USUALLY you want to commit lock files. I assume you're talking about a rakelock here

Reply:
that pattern is for the .docx lock, it should go in